src/core/graph.js

```diff
@@ -1212,3 +1212,73 @@ export function edgeToParts(
   const timestampMs = edge.timestampMs;
   return {addressParts, srcParts, dstParts, timestampMs};
 }
+
+export function compareGraphs(
+  firstGraph: Graph,
+  secondGraph: Graph
+): {|
+  +isEqual: boolean,
```

Comment:
nit: `isEqual` feels a little odd as a name, since the graph comparison is an object describing two graphs. so `areEqual` might be the right conjugation? or the even more explicit `graphsAreEqual`?

also, did you know that there's a (more efficient) `graph.equals(otherGraph)` method already defined? it could make sense for callers to check graph equality using that method, and then compute the (more expensive) full comparison result only when the graphs are not equal, in which case this field might not be as necessary. that said, having this field does seem like a nice convenience, so not requesting it be removed.

Reply:
done and done